lib/main.dart

```diff
@@ -1,13 +1,18 @@
 import 'package:firebase_core/firebase_core.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_ecommerce/firebase_options.dart';
+import 'package:flutter_ecommerce/utilities/constants/theme_manager.dart';
+import 'package:form_builder_validators/form_builder_validators.dart';
 import 'package:flutter_ecommerce/services/auth.dart';
 import 'package:flutter_ecommerce/utilities/router.dart';
 import 'package:flutter_ecommerce/utilities/routes.dart';
 import 'package:provider/provider.dart';
 
 Future<void> main() async {
   WidgetsFlutterBinding.ensureInitialized();
-  await Firebase.initializeApp();
+  await Firebase.initializeApp(
+    options: DefaultFirebaseOptions.currentPlatform,
```

Comment:
We didn't add the firebase options yet, but that's an excellent suggestion.
We will add this in the future because we don't have one centralized firebase_options file.
Everyone has their own firebase config settings